### ISSUE TITLE
Access Request reviewers should receive a notification email

### ISSUE THREAD

**Issue by jon-can** [gravitational/teleport#6438]
### What
As of Teleport 6.1 you can specify reviewers when creating an access request. Teleport should be able to email these reviewers to alert them about the access request.

### How
Add an area to specify SMTP settings?

**Comment by benarent** [comment]
I would suggest we do this in the form of a plugin. That'll integrate with a range of cloud-based email providers.

**Comment by wadells** [comment]
We have an access request plugin for email available: https://github.com/gravitational/teleport-plugins/tree/master/access/email

However, I do see that this plugin is missing from the workflows documentation:

https://github.com/gravitational/teleport/tree/8c2ced7e2bd14e6456b2eedf55c3b92a9242396e/docs/pages/enterprise/workflow

I'd expect we'd see a `ssh-approval-email.mdx`

**Comment by zmb3** [comment]
@nklaassen maybe this is something that can be addressed when we add support to plugins for resource access requests.

**Comment by nklaassen** [comment]
yeah, as walt said the email plugin exists but docs are missing, I can add when updating for resource access requests

**Comment by wadells** [comment]
I found a PR open to add the docs: https://github.com/gravitational/teleport/pull/8695 (thanks @stevenGravy!)  We can run with that or implement from scratch.

I'd hoped to finish Steven's PR last week, but got distracted.  The work is up for grabs -- I make no claim on getting it done soon.
